### PR TITLE
new: add serviceType field in connectionexceptionreport

### PR DESCRIPTION
**Pull request by sibicramesh**

### CODE REVIEW

File: specs/connectionexceptionreport.spec [hunk 1]
```diff
@@ -128,6 +128,20 @@ attributes:
...
+    exposed: true
+    stored: true
+    allowed_choices:
+    - L3
+    - HTTP
+    - TCP
+    default_value: L3
+    omit_empty: true
+    extensions:
+      bson_name: o
```
Comment (aaslamin) [inline specs/connectionexceptionreport.spec:143]: This should be `l` now.

**Review by aaslamin — CHANGES_REQUESTED** [review]

File: specs/connectionexceptionreport.spec (hunk 1 above)
Comment (sibicramesh) [reply specs/connectionexceptionreport.spec:143]: It should be `k` ? no ?

File: specs/connectionexceptionreport.spec [hunk 2]
```diff
@@ -128,14 +128,28 @@ attributes:
...
+
   - name: sourceIP
     description: Source IP address.
     type: string
     exposed: true
     stored: true
     omit_empty: true
     extensions:
-      bson_name: k
+      bson_name: l
```
Comment (aaslamin) [inline specs/connectionexceptionreport.spec:152]: This is backwards **incompatible** as the database would have already used the previous field name for serialization.

You can't change the `bson_name` already assigned. It's the equivalent of changing the field name as far as the database is concerned.

File: specs/connectionexceptionreport.spec [hunk 3]
```diff
@@ -152,7 +166,7 @@ attributes:
     example_value:
     - Unknown
     extensions:
-      bson_name: l
+      bson_name: m
```
Comment (aaslamin) [inline specs/connectionexceptionreport.spec:169]: This is backwards **incompatible** as the database would have already used the previous field name for serialization.

You can't change the `bson_name` already assigned. It's the equivalent of changing the field name as far as the database is concerned.

File: specs/connectionexceptionreport.spec [hunk 4]
```diff
@@ -172,4 +186,4 @@ attributes:
     example_value: 1
     omit_empty: true
     extensions:
-      bson_name: "n"
+      bson_name: o
```
Comment (aaslamin) [inline specs/connectionexceptionreport.spec:189]: This is backwards **incompatible** as the database would have already used the previous field name for serialization.

You can't change the `bson_name` already assigned. It's the equivalent of changing the field name as far as the database is concerned.

**Review by aaslamin — CHANGES_REQUESTED** [review]

File: specs/connectionexceptionreport.spec (hunk 1 above)
Comment (aaslamin) [reply specs/connectionexceptionreport.spec]: `k` is already used by [`sourceIP`](https://github.com/PaloAltoNetworks/gaia/blob/fefc6a112ebb9232dc3a495ba64ca822c4e938e4/specs/connectionexceptionreport.spec#L138).

Comment (aaslamin) [reply specs/connectionexceptionreport.spec]: My bad, the diff didn't make it easy to catch this, but looking at the model, `o` was indeed the correct name to use for the next field name.

Comment (sibicramesh) [reply specs/connectionexceptionreport.spec:143]: Np. Fixed.

**Review by aaslamin — APPROVED** [review]
LGTM :shipit:

**Pull request merged**